start-deployFabric

```diff
@@ -63,12 +63,12 @@ if [[ ! -e $installMarker ]]; then
   done
   set -e
   if (($tries < 0)); then
-    echo "Fabric failed to install after several tries." >&2
+    log "Fabric failed to install after several tries." >&2
     exit 10
   fi
   export SERVER=fabric-server-launch.jar
-  echo "Using server $SERVER"
-  echo $SERVER > $installMarker
+  log "Using server $SERVER"
+  log $SERVER > $installMarker
```

Comment:
This needs to remain an `echo`

Reply:
I'm assuming you only want to change line 71?

```suggestion
  echo $SERVER > $installMarker
```

Reply:
Thanks for checking. Yes, just line 71